src/main/java/seedu/duke/exercisegenerator/GenerateExercise.java

```diff
@@ -125,30 +117,33 @@ public ArrayList<ExerciseData> generateSpecificDifficultySet(String difficultyLe
     }
 
     public ArrayList<ExerciseData> generateSetAll() {
-        ArrayList<ExerciseData> allExerciseList = exerciseData;
-        return allExerciseList;
+        return this.exerciseDataList;
     }
 
     private static String parseDifficultyLevel(String difficulty) throws DukeError {
-        if (difficulty.equals(EASY)) {
+        switch (difficulty) {
```

Comment:
Changing from if-else statements to case statements makes code look neater, good job